src/content/docs/en/community-resources/content.mdx

```diff
@@ -77,9 +78,15 @@ Have you written a recipe for Astro? [Edit this page](https://github.com/withast
 - [Add searching to your site with Pagefind](https://blog.otterlord.dev/posts/astro-search/)
 - [Add searching to your site with Fuse.js](https://www.youtube.com/watch?v=XnV_2MWqAhQ)
 - [Add a comments section to your Astro blog using Giscus](https://elazizi.com/posts/add-comments-section-to-your-astro-blog/)
+- [Using GreenSock Animation Platform (GSAP) in Astro](https://www.launchfa.st/blog/gsap-astro)
+- [Using GreenSock Animation Platform (GSAP) in Astro with View Transitions](https://www.launchfa.st/blog/gsap-astro-view-transitions)
 ### Internationalization and Localization
 - [How to make your Astro website multilingual with Crowdin: Astro localization guide](https://crowdin.com/blog/2023/06/21/astro-localization-and-i18n)
 - [Translate routes in Astro for content collections and subpages](https://www.webdesign-sopelnik.de/en/blog/translate-routes-for-astro-content-collections-or-subpages-with-trailingslash-and-base-support/)
 ### Astro DB
 - [Creating a Guestbook with Astro DB](https://ryantrimble.com/blog/creating-a-guestbook-with-astro-db/)
 - [How to Build a Views Counter with Astro DB](https://www.vorillaz.com/astro-views)
+### Deployment Platforms
```

Comment:
What about moving these to the appropriate deploy guides, since we already have those?